sql-plugin/src/main/spark330db/scala/com/nvidia/spark/rapids/shims/GpuBatchScanExec.scala

```diff
@@ -43,7 +40,7 @@ case class GpuBatchScanExec(
     ordering: Option[Seq[SortOrder]], 
     @transient table: Table,
     reusesFileListingResultsSourceNode: Option[BatchScanExec])
-    extends DataSourceV2ScanExecBase with GpuBatchScanExecMetrics {
+    extends GpuBatchScanExecBase(scan, runtimeFilters) {
   @transient lazy val batch: Batch = scan.toBatch
 
   // All expressions are filter expressions used on the CPU.
```

Comment:
NIT: Since now `GpuBatchScanExecBase` is introduced,  we can move more common into the base class. e.g. `gpuExpressions`.